tests/src/Core/Storage/StorageConfigTest.php

```diff
@@ -19,17 +19,17 @@
  *
  */
 
-namespace Friendica\Test\src\Model\Storage;
+namespace Friendica\Test\src\Core\Storage;
 
-use Friendica\Model\Storage\IStorageConfiguration;
+use Friendica\Core\Storage\Capability\ICanConfigureStorage;
 use Friendica\Test\MockedTest;
 
 abstract class StorageConfigTest extends MockedTest
 {
-	/** @return IStorageConfiguration */
+	/** @return \Friendica\Core\Storage\Capability\ICanConfigureStorage */
 	abstract protected function getInstance();
 
-	abstract protected function assertOption(IStorageConfiguration $storage);
+	abstract protected function assertOption(\Friendica\Core\Storage\Capability\ICanConfigureStorage $storage);
```

Comment:
Extraneous absolute name:
```suggestion
	abstract protected function assertOption(ICanConfigureStorage $storage);
```